src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -158,6 +161,13 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
 
       <Box id="scrollTo--artistAuctionResultsTop" />
 
+      <ThemeProviderV3>
```

Comment:
No need for a ThemeProvider here, as we set the theme [in our routes file](https://github.com/artsy/force/blob/a8203e756fcc0562d2d7aa35a2a210d7692bab43/src/v2/Apps/Artist/artistRoutes.tsx#L181)